pennylane/templates/embeddings.py

```diff
@@ -123,10 +123,15 @@ def AngleEmbedding(features, wires, rotation='X'):
         raise ValueError("Rotation has to be `X`, `Y` or `Z`; got {}.".format(rotation))
 
 
-def AmplitudeEmbedding(features, wires):
+def AmplitudeEmbedding(features, wires, pad=False, normalize=False):
     r"""Encodes :math:`2^n` features into the amplitude vector of :math:`n` qubits.
 
-    The absolute square of all elements in ``features`` has to add up to one.
+    If the total number of features to embed are less than the :math:`2^n` available amplitudes,
+    non-informative constants (zeros) can be padded to ``features``. To enable this, the argument
+    ``pad`` should be set to ``True``. It is set to ``False`` by default.
```

Comment:
Sphinx will automatically list the default value of optional arguments, so probably no need to specify here.
```suggestion
    ``pad`` should be set to ``True``.
```